spec/features/user_session_timeout_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'features/support/sign_up_helpers'
+require 'features/support/not_signed_in'
+require 'timecop'
+
+describe 'with devise timeoutable enabled' do
+  after { Timecop.return }
+
+  context 'when user has not signed in' do
+    before { visit root_path }
+
+    it_behaves_like 'not signed in'
+  end
+
+  describe 'user sessions timeout' do
```

Comment:
I wouldn't include this wrapping describe block, I don't think it helps to describe what we're testing better.